src/style.rs

```diff
@@ -69,15 +69,15 @@
 //! ```no_run
 //! use std::io::{stdout, Write};
 //!
-//! use crossterm::{execute, Result, Output};
+//! use crossterm::{execute, Result, style::Print};
 //! use crossterm::style::{SetAttribute, Attribute};
 //!
 //! fn main() -> Result<()> {
 //!     execute!(
 //!         stdout(),
 //!         // Set to bold
 //!         SetAttribute(Attribute::Bold),
-//!         Output("Styled text here.".to_string()),
+//!         Print("Styled text here.".to_string()),
```

Comment:
Isn't "Styled text here" a bit misleading here? Sounds like PrintStyledContent. Maybe "Bold text" will be better?